SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKSimulatedPlayerDeployment.cpp

```diff
@@ -728,8 +728,11 @@ void SSpatialGDKSimulatedPlayerDeployment::OnPrimaryDeploymentNameCommited(const
 	}
 	DeploymentNameInputErrorReporting->SetError(TEXT(""));
 	
-	USpatialGDKEditorSettings* SpatialGDKSettings = GetMutableDefault<USpatialGDKEditorSettings>();
-	SpatialGDKSettings->SetPrimaryDeploymentName(InputDeploymentName);
+	USpatialGDKEditorSettings* SpatialGDKEditorSettings = GetMutableDefault<USpatialGDKEditorSettings>();
+	SpatialGDKEditorSettings->SetPrimaryDeploymentName(InputDeploymentName);
+
+	// Also overwrite DevelopmentDeploymentToConnect in SpatialGDKEditorSettings.(Request by UNR-3482)
+	SpatialGDKEditorSettings->SetDevelopmentDeploymentToConnect(InputDeploymentName);
```

Comment:
Can we copy this when the deployment is being launched? Otherwise someone can edit this field and not start a deployment, but the deployment to connect would have changed.